lib/fxpay/index.js

```diff
@@ -69,14 +69,13 @@
     }
 
     var partialProdInfo = {productId: productId};
-    var log = settings.log;
 
     if (!onPurchase) {
       onPurchase = function _onPurchase(err, returnedProdInfo) {
         if (err) {
           throw err;
         }
-        console.log('product', returnedProdInfo.productId, 'purchased');
+        settings.log('product', returnedProdInfo.productId, 'purchased');
```

Comment:
Oh, it's because there's no way I can think of to test this. The test needs to define a callback to know when done.